program/rust/src/c_oracle_header.rs

```diff
@@ -34,94 +34,137 @@ pub trait PythAccount: Pod {
     }
 }
 
-impl PythAccount for pc_map_table_t {
+impl PythAccount for MappingAccount {
     const ACCOUNT_TYPE: u32 = PC_ACCTYPE_MAPPING;
     const INITIAL_SIZE: u32 = PC_MAP_TABLE_T_PROD_OFFSET as u32;
```

Comment:
Can we not use `size_of::<MappingAccount>` here as in `ProductAccount`?

Reply:
No, added a comment explaning.